libs/astx/src/astx/callables.py

```diff
@@ -388,6 +388,47 @@ def get_struct(self, simplified: bool = False) -> ReprStruct:
         return self._prepare_struct(key, value, simplified)
 
 
+@public
+@typechecked
+class YieldStmt(StatementType):
+    """AST class for yield statement."""
+
+    value: Optional[Expr]
+
+    def __init__(
+        self,
+        value: Optional[Expr] = None,
+        loc: SourceLocation = NO_SOURCE_LOCATION,
+        parent: Optional[ASTNodes] = None,
+    ) -> None:
+        """Initialize the YieldStmt instance.
+
+        Args:
```

Comment:
for now we don't need to use arguments in the docstring. we will use another thing in the future for that.
btw, we use numpydocs convention. so for now just keep the dostring title "Initialize the YieldStmt instance."

this is a small issue, so I will merge this PR, you can fix that in a follow up pr.